### DISCUSSION TITLE
Is there any way to open the program with a preloaded playlist?

### DISCUSSION THREAD

**Discussion by caiota** [FreeTubeApp/FreeTube#4840] [Q&A] [answered]
Is there any way to open the program with a preloaded playlist?

like
C:\Users\yourself\AppData\Local\Programs\FreeTube\FreeTube.exe -playlist=https://www.youtube.com/playlist...

**Comment by Areiuna** [comment]
how do u watch sum on here pls pls help me

**Comment by absidue** [comment, marked as answer]
@caiota You can pass the URL directly, e.g. `C:\Users\yourself\AppData\Local\Programs\FreeTube\FreeTube.exe https://www.youtube.com/playlist...`
